__std__/nix/plutus/devshells/plutus-shell.nix

```diff
@@ -1,10 +1,19 @@
 { inputs, cell }:
 
+let
+  inherit (inputs.cells.toolchain.library) pkgs;
+in
+
 inputs.std.std.lib.mkShell {
 
   name = "plutus-shell";
 
   imports = [
     inputs.cells.toolchain.devshellProfiles.haskell
   ];
+
+  packages = pkgs.lib.optionals (!pkgs.stdenv.isDarwin) [
```

Comment:
It's very unclear which things go here versus in the profile. I wonder if using profiles at this stage is overkill when we don't have more than one shell? Or at least, we should have some clear guidelines for where to put what.